docs-site/content/guide/semantic-search.md

```diff
@@ -442,6 +442,51 @@ Notice how searching for `Desktop copier` returns `Desktop` as a result which is
 }
 ```
 
+
+### Pagination
+
+For effective pagination with semantic or hybrid search, use the `k` parameter in `vector_search` to ground the search to a specific number of closest items:
+
+```shell{11}
+curl 'http://localhost:8108/multi_search' \
+  -H "X-TYPESENSE-API-KEY: ${TYPESENSE_API_KEY}" \
+  -X POST \
+  -d '{
+    "searches": [
+      {
+        "q": "device to type things on",
+        "query_by": "embedding",
+        "collection": "products",
+        "prefix": "false",
+        "vector_query": "embedding([], k: 200)",
+        "exclude_fields": "embedding",
+      }
+    ]
+  }'
+```
+
+Optionally, use the `distance_threshold` parameter in `vector_query` to fine-tune semantic search results:
```

Comment:
This feels like a separate section. Let's add another section heading here, instead of including it under pagination.

Reply:
> This feels like a separate section. Let's add another section heading here, instead of including it under pagination.


Section added on https://github.com/typesense/typesense-website/pull/259/commits/a674f7a546f7388a1314f6296c3fe37d7a174c2c